
Remove libanl from list of linked libraries as it is not needed at all
and does not exist on musl.

--- libs-e5c53d648f3c4694385bbe488e7d47eaa36c229a/userspace/libsinsp/CMakeLists.txt~	2022-03-25 09:50:03.000000000 +0000
+++ libs-e5c53d648f3c4694385bbe488e7d47eaa36c229a/userspace/libsinsp/CMakeLists.txt	2024-04-13 13:40:53.222240196 +0000
@@ -230,8 +230,7 @@ if(NOT WIN32)
 
 			if(NOT MUSL_OPTIMIZED_BUILD)
 				list(APPEND SINSP_LIBRARIES
-				rt
-				anl)
+				rt)
 			endif()
 
 		else()
